src/registerServiceWorker.ts

```diff
@@ -0,0 +1,114 @@
+// tslint:disable:no-console
```

Comment:
Can we delete this file?

Reply:
Probably, it just was dumped there by create-react-app with ts enabled, but i don't think we're using it